Makefile

```
@@ -108,8 +111,11 @@ go-test-api: check-env
 	@pushd api && ../scripts/test-go.sh && popd
 
 mod-check: check-env
-	@echo 'running: go mod verify'
-	@go mod verify && [ "$(shell sha512sum go.mod)" = "`sha512sum go.mod`" ] || ( echo "ERROR: go.mod was modified by 'go mod verify'" && false )
+	for module in $(GO_MODULES_LIST); do \
+		echo "running: go mod checks in $$module"; \
+		(cd "$$module" && go mod verify && go mod tidy && go mod vendor);\
```

```suggestion
(cd "$$module" go mod vendor && go mod tidy  && go mod verify && );\
```

Lets change the order to run verify at last

---

> Lets change the order to run verify at last

done, ptal